packages/flutter/lib/src/widgets/editable_text.dart

```diff
@@ -1346,6 +1347,14 @@ class EditableText extends StatefulWidget {
   /// than 1.
   final ScrollBehavior? scrollBehavior;
 
+  /// Optionally, the parent build context if this editable text was built as part of
+  /// a larger text field.
+  ///
+  /// If this is provided then focus will be attached to the parent context instead of
+  /// the editable text. If the [parentContext] is later modified while the editable text
```

Comment:
Maybe also explain what's the different consequences to attach to this context vs parent context, and why developer may want to do that